frontend/src/pages/Login.js

```diff
@@ -1,23 +1,45 @@
-import React from 'react'
-import { Center, Heading, VStack } from '@chakra-ui/react'
+import React, { useState } from 'react'
```

Comment:
revert all changes in this file, it was refactored in a different pr, I'm assuming this change was introduced accidentally when you handled merge conflicts